commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemServiceImpl.java

```diff
@@ -147,6 +147,34 @@ public CommerceInventoryWarehouseItem getCommerceInventoryWarehouseItem(
 				commerceInventoryWarehouseId, start, end);
 	}
 
+	@Override
+	public List<CommerceInventoryWarehouseItem>
+			getCommerceInventoryWarehouseItemsByCompanyId(
```

Comment:
We could name this just `getCommerceInventoryWarehouseItems`, right? That'd be a better name -- we shouldn't specify `*ByX` unless we have to do differentiate from other methods with the same signature.

Reply:
We already have a method with that signature